### DISCUSSION TITLE
How to change url text to the actual page title?

### DISCUSSION THREAD

**Discussion by fp-jcorriveau** [nfrasser/linkifyjs#453] [General]
Is there an options to turn the url text as the actual page title of the url...
For exemple, if in my text, I have https://google.com/

I would like it to be ![image](https://github.com/Hypercontext/linkifyjs/assets/53450693/6e7f8443-53a0-4f45-9cf5-5108bc94fc9a)
(JIRA Style) With the favicon and title of the link.

**Comment by nfrasser** [comment]
Hi @fp-jcorriveau, check out the `render` option: https://linkify.js.org/docs/options.html#render

You can use this to output custom HTML based on a the value of a given link (e.g., an `<img>` tag instead of `<a>` if the `href` attribute contains text "google.com")

**Comment by chacepro** [comment]
I am doing this same thing now and I have it working by making an asynchronous call to a custom lambda function. It returns how I want it to, however, the way I have it setup in React if there are more than one string that fits a "URL" for Linkify it toggles between the two. I am struggling to find a way to have the render contained to the individual elements.

**Reply by chacepro** [reply]
See this discussion and see if that's what you're after @fp-jcorriveau 
[Discussion](https://github.com/Hypercontext/linkifyjs/discussions/489)
